harness/apisupport.harness/release/README

```diff
@@ -1,1363 +1,1363 @@
-INTRODUCTION
-------------

```

Comment:
The diff shouldn't remove everything and add it again. Try to fix your line endings.